servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ConnectionCloseHeaderHandlingTest.java

```diff
@@ -338,14 +359,13 @@ public void serverCloseTwoPipelinedRequestsSentBeforeFirstResponse() throws Exce
             assertResponsePayloadBody(response);
 
             awaitConnectionClosed();
-            // FIXME: temporary disable check for /second until https://github.com/apple/servicetalk/pull/1141
```

Comment:
can we revert this portion now that we have the `@Ignore`?

Reply:
(same comment below)

Reply:
Originally, I thought only the last check is causing the issue. But actually, the response payload body for the first request may also fail.
Decided to uncomment specific assertions and ignore the whole test instead until we have #1141.